docs/feature/storage-backends/helm.mdx

```diff
@@ -61,8 +61,9 @@ import TabItem from '@theme/TabItem';
 
 ```bash
 cat > /tmp/helm-storage-input-params.yaml << ENDOFFILE
-input-parameters:
- version: 0.6.0-5f348f9 # optional
+input-parameters: {}
+ # Optional input parameters:
```

Comment:
```suggestion
 # Optional input parameters. e.g.:
```

I'm not sure if the version is needed if this is the "default" approach - but the comment doesn't break anything, so I would rather keep it 🙂  If you think otherwise, that's fine, you can completely delete the lines 65 and 66.